webapp/package-lock.json

```diff
@@ -4048,6 +3958,17 @@
         "npm": ">=7.0.0"
       }
     },
+    "channels/node_modules/@types/react-select": {
+      "version": "3.0.19",
+      "resolved": "https://registry.npmjs.org/@types/react-select/-/react-select-3.0.19.tgz",
+      "integrity": "sha512-d+6qtfFXZeIOAABlVL1e50RZn8ctOABE4tFDxM6KW4lKuXgTTgLVrSik5AX9XjBjV7N80FtS6GTN/WeoXL9Jww==",
+      "dev": true,
+      "dependencies": {
+        "@types/react": "*",
+        "@types/react-dom": "*",
+        "@types/react-transition-group": "*"
+      }
+    },
```

Comment:
Reinstalling `@types/react-select@3.0.19` after `global-style` was the latest whack-a-mole which seems to stabilize it (types and unit tests are passing with all workspaces together)